src/nl/hannahsten/texifyidea/action/wizard/graphic/InsertGraphicWizardAction.kt

```diff
@@ -51,8 +51,12 @@ class InsertGraphicWizardAction(private val initialFile: File? = null) : AnActio
 
     /**
      * Opens and handles the graphic insertion wizard.
+     * // todo method name does not match, what does it do?
+     * // todo why are we adding a method about pasting images when the PR is about styled text?
```

Comment:
Styled text, or html, can contain images, and so the way we currently do things is pick a single paste provider based on clipboard content. This will not work with "styled text" and images in the same paste, hence the two need to be merged in order to not mess up the paste

Reply:
That is what I was trying to do, yes. It runs through the html choosing the correct provider for each element